docs/_articles/en/v3.x/salesforce-graph-engine/working-with-sfge.md

```diff
@@ -123,12 +123,12 @@ public Integer foo(String input) {
 **Result:** A Graph Engine analysis attempt on this code path results in a User Action Violation on this path. Analysis on other paths can proceed. Sometimes other violations are returned. This message is returned: ```Rename or delete this reused variable to proceed with the analysis```.
 
 
-## Understand OutOfMemory Errors
+## Understand LimitReached Errors
 
-When Graph Engine analyzes highly complex code, it runs out of heap space, which  results in an `OutOfMemory` error. To decrease the occurrence of `OutOfMemory` errors and to complete as much analysis as possible within a shorter period, we added processing limits on Graph Engine. These limits help Graph Engine to fail fast when a path’s analysis is approaching an `OutOfMemory` error. This fail-fast process includes preemptively aborting a path analysis when Graph Engine encounters a path that’s too complex.
+When Graph Engine analyzes highly complex code, it runs out of heap space, which  results in an `LimitReached` error. To decrease the occurrence of `LimitReached` errors and to complete as much analysis as possible within a shorter period, we added processing limits on Graph Engine. These limits help Graph Engine to fail fast when a path’s analysis is approaching an `LimitReached` error. This fail-fast process includes preemptively aborting a path analysis when Graph Engine encounters a path that’s too complex.
```

Comment:
There are a few "an `LimitReached`"'s, which should be "a `LimitReached`" here and at line 131.